app/api/routers/breeze_buddy/daily.py

```
@@ -0,0 +1,108 @@
+        # 3. Prepare runner arguments for Daily transport
+        runner_args = DailyRunnerArguments(
+            room_url=room_url,
+            token=bot_token,
+            body={
+                "call_sid": call_sid,
+                "session_id": session_id,
+            },
+        )
+
+        # 4. Start the bot in background
+        asyncio.create_task(bot(runner_args))
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)